src/main/java/tr/edu/ogu/ceng/service/InternshipService.java

```diff
@@ -32,12 +32,6 @@
 public class InternshipService {
 	@Autowired
 	private InternshipRepository internshipRepository;
-	private StudentRepository studentRepository;
```

Comment:
Buradaki repo dosyalarının silinmemesi gerekiyor eğer silinirse internship test kısmında hata çıkar ve kod compile error verir

Reply:
Bütün repo dosyaları için geçerli bu durum